app/helpers/sales_helper.rb

```diff
@@ -0,0 +1,6 @@
+module SalesHelper
+  def in_dollars(sale)
```

Comment:
Nice! Excellent extraction of this logic to somewhere other than the views/controllers